account_banking_ach_direct_debit_portal/views/autopay_rules_templates.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="portal_autopay_rules">
+        <t t-call="portal.portal_layout">
+            <t t-set="breadcrumbs_searchbar" t-value="True" />
+
+            <t t-call="account_banking_ach_direct_debit_portal.debit_portal_searchbar">
+                <t t-set="title">Autopay Rules</t>
+            </t>
+
+            <div id="autopay-settings">
+                <div class="d-flex justify-content-between align-items-center mb-3">
+                    <div class="d-flex justify-content-start align-items-center">
+                        <h4 class="header-title mb-0 me-2">Autopay Rules</h4>
+
+                        <div class="text-danger">
+                            <i
+                                class="fa fa-exclamation-circle me-1"
+                            /> Must required bank authentication
+                        </div>
+                    </div>
+
+                    <label class="btn btn-toggle">
+                      Disable Autopay
+                      <input type="checkbox" />
+                      <span class="slider" />
+                    </label>
+                </div>
+
+                <label class="rule-option" for="rule_1">
+                    <div
+                        class="d-flex justify-content-between align-items-center cursor-pointer w-100"
+                    >
+                        <div class="d-flex justify-content-start align-items-center">
+                            <input
+                                type="radio"
+                                name="autopay_rule"
+                                id="rule_1"
+                                checked="true"
+                            />
+                            <span class="ms-2">Number of days before month end</span>
+                        </div>
+
+                        <span
+                            class="note text-muted"
+                        >5 days before month end ( Default )</span>
+                    </div>
```

Comment:
there should an input here to defined the number of days
5 days is a default (placeholder)

Reply:
still no input?

also should be a new field on partner, not the same for the whole odoo database